package/setup.py

```diff
@@ -446,9 +501,24 @@ def dynamic_author_list():
     ]
 
     config = Config()
-    exts, cythonfiles = extensions(config)
+    require_cython = ['Cython (>=0.16)', 'Cython>=0.16'] if is_dev else []
+    # dev installs must build their own cythonized files.
+    if is_dev:
+        use_cython = True
+    else:
+        use_cython = config.get('use_cython', default=False)
+    if use_cython:
+        print('Will attempt to use Cython.')
+        if not cython_found and is_release:
+            # dev installs might not have cython straight away.
+            print("Couldn't find a Cython installation. "
```

Comment:
It would come up if you were installing from master and set `MDA_USE_CYTHON`. In any case as per your suggestion, I'm no longer raising this error and instead take care of installing cython in this case.